dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/service/ContributionService.java

```diff
@@ -1,43 +1,41 @@
-        //
+        Map<Integer,String> failedContributions = new HashMap<>();
+        // get all the values to process via maat call
+        contributionsList = contributionClient.getContributions("ACTIVE");
```

Comment:
Just removing the placeholder data that was populating. Replacing with actual call.